lib/internal/Magento/Framework/Communication/Config/Reader/XmlReader.php

```diff
@@ -19,4 +23,38 @@ class XmlReader extends \Magento\Framework\Config\Reader\Filesystem
         '/config/topic' => 'name',
         '/config/topic/handler' => 'name'
     ];
+
+    /**
+     * @param FileResolverInterface $fileResolver
+     * @param XmlReader\Converter $converter
+     * @param XmlReader\SchemaLocator $schemaLocator
+     * @param ValidationStateInterface $validationState
+     * @param string $fileName
+     * @param array $idAttributes
+     * @param string $domDocumentClass
+     * @param string $defaultScope
+     * phpcs:disable Generic.CodeAnalysis.UselessOverridingMethod
```

Comment:
`phpcs:ignore` works on documented scope - eg. if you add `phpcs:ignore` to the class - it's being ignored in whole class, if it's applied to method - it's applied to method only.